### DISCUSSION TITLE
Filter Branches is case sensitive

### DISCUSSION THREAD

**Discussion by unomonteiro** [community/community#21195] [Other Feature Feedback, Questions, & Ideas]
When searching for branches with "MVI", if I filter for "mvi" nothing appears. only if I search for "MVI".

**Comment by look** [comment]
@unomonteiro hi! Can you clarify where in the GitHub UX you are searching for branches? Thanks.

**Comment by unomonteiro** [comment]
Hell @look here are some screenshots:
we can see there is a "dagger" branch
<img width="796" alt="Screenshot 2022-07-25 at 19 12 10" src="https://user-images.githubusercontent.com/7604348/180845713-5ac973b6-c7b7-43f8-8c52-a866d7c4a1b0.png">

Search for "Dagger" nothing appears
<img width="796" alt="Screenshot 2022-07-25 at 19 12 25" src="https://user-images.githubusercontent.com/7604348/180845718-d2b68271-51c3-4763-9c74-d1cbdd78ba0e.png">

**Reply by look** [reply]
Thanks for clarifying. I've recategorized this post because the `Code Search and Navigation` category is about code search ([cs.github.com](https://cs.github.com/home)) and code navigation (jump to definition, find references).
